deepsubtree_test.go

```diff
@@ -125,3 +125,106 @@ func TestDeepSubtreeWithUpdates(t *testing.T) {
 		require.Equal(dst.root.hash, tree.root.hash)
 	}
 }
+
+// Tests adding and deleting keys in the deepsubtree returns the
+// correct roots
+func TestDeepSubtreeWWithAddsAndDeletes(t *testing.T) {
+	require := require.New(t)
+	getTree := func() *MutableTree {
+		tree, err := getTestTree(5)
+		require.NoError(err)
+
+		tree.Set([]byte("b"), []byte{2})
+		tree.Set([]byte("a"), []byte{1})
+
+		_, _, err = tree.SaveVersion()
+		require.NoError(err)
+		return tree
+	}
+	tree := getTree()
+	fmt.Println("PRINT TREE")
+	_ = printNode(tree.ndb, tree.root, 0)
+	fmt.Println("PRINT TREE END")
```

Comment:
Don't use Println in tests. Use t.Log.

Reply:
Removed Println in tests as other parts of the codebase do: https://github.com/celestiaorg/iavl/blob/9511ba01120acfea75cd903886a55210b80047c8/tree_test.go#L1189